buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/classycle/Classycle.kt

```diff
@@ -40,29 +41,27 @@ import javax.inject.Inject
 
 
 @CacheableTask
-open class Classycle : DefaultTask() {
+open class Classycle @Inject constructor(
+    @get:Internal val classesDirs: FileCollection,
+    excludePatterns: Provider<List<String>>,
+    @get:Input val reportName: String,
+    @get:Internal val reportDir: File,
+    reportResourcesZip: Provider<RegularFile>,
+    @get:Internal private val antBuilder: IsolatedAntBuilder
```

Comment:
I should not be necessary to annotate the `private val`.

Reply:
done